src/plugins/dashboard/public/dashboard_strings.ts

```diff
@@ -388,6 +388,11 @@ export const dashboardLoadingErrorStrings = {
       defaultMessage: 'Error encountered while loading saved dashboard: {message}',
       values: { message },
     }),
+  getDashboardGridError: (message: string) =>
+    i18n.translate('dashboard.loadingError.dashboardGridErrorMessage', {
+      defaultMessage: 'Unable to load dashboard: {message}',
+      values: { message },
+    }),
```

Comment:
I moved this string to no longer be inline where it was used in `dashboard_grid` 👍